elements/barcode-reader/src/barcode-reader.js

```diff
@@ -285,6 +284,27 @@ class BarcodeReader extends LitElement {
       videoElement.srcObject = stream;
     }
 
+    this.shadowRoot.querySelector(".render").addEventListener("click", () => {
+      if (
+        this.shadowRoot.querySelector(".render").innerHTML ===
+        "Show scanner"
```

Comment:
This is a poor method of toggling. Use a boolean value that isn't mapped to `properties` and then toggle that value. When the state of that changes, THEN you run your code associated w/ state change.